xml/admin_monitoring_alerting.xml

```diff
@@ -1752,14 +1741,13 @@ ceph-volume lvm create --osd-id $N --data /path/to/device</screen>
     <varlistentry>
       <term><option>SMALLER_PGP_NUM</option></term>
       <listitem>
-        <para>One or more pools has a <option>pgp_num</option> value less than <option>pg_num</option>.
-          This is normally an indication that the PG count was increased
-          without also increasing the placement behavior.
-          This is sometimes done deliberately to separate out the split step
-          when the PG count is adjusted from the data migration that is needed
-          when <option>pgp_num</option> is changed.
-          This is normally resolved by setting <option>pgp_num</option> to
-          match <option>pg_num</option>, triggering the data migration, with:
+        <para>One or more pools have a <option>pgp_num</option> value less than <option>pg_num</option>,
```

Comment:
I recenty added a `Warning` elsewhere to address precisely this. May be worth an xlink? Up to you.